config/provisioners/natss/README.md

```diff
@@ -0,0 +1,47 @@
+# NATS Streaming Channels
+
+
+### Deployment steps:
+
+1. Setup [Knative Eventing](../../../DEVELOPMENT.md).
+1. Apply the 'natss' ClusterChannelProvisioner, Controller, and Dispatcher.
+     ```shell
+     ko apply -f config/provisioners/natss/
```

Comment:
The above would provision natss, and the cpp - right ?

I think I'd prefer some separation here (e.g see Kafka). and that the actual provisioning of the natss bits would be more a sample - as well as showing how to use existing/external natss.

and than for the `ccp`, have just something like:
```
ko apply -f config/provisioners/natss/natss.yaml
```

Reply:
Done